webextensions/api/downloads.json

```diff
@@ -150,12 +158,20 @@
                   "version_added": "30"
                 },
                 "edge": "mirror",
-                "firefox": {
-                  "version_added": "47"
-                },
+                "firefox": [
+                  {
+                    "version_added": "69"
+                  },
+                  {
+                    "version_added": "47",
+                    "version_removed": "69",
+                    "notes": "Always returns `undefined`."
+                  }
+                ],
                 "firefox_android": {
                   "version_added": "48",
-                  "version_removed": "79"
+                  "version_removed": "79",
```

Comment:
Was this only fixed later in Android?

Reply:
There were no Firefox for Android releases 69 through 78, so I never received the fix.